ops/docker/deployment-utils/Dockerfile

```diff
@@ -34,10 +34,30 @@ COPY --from=base /root/.foundry/bin/cast /usr/local/bin/cast
 COPY --from=base /root/.foundry/bin/anvil /usr/local/bin/anvil
 COPY --from=go-base /go/bin/dasel /usr/local/bin/dasel
 
-# Install geth
-RUN curl -L https://gethstore.blob.core.windows.net/builds/geth-linux-amd64-1.15.11-36b2371c.tar.gz \
-    -o geth.tar.gz \
- && echo "a14a4285daedf75ea04a7a298e6caa48d566a2786c93fc5e86ec2c5998c92455  geth.tar.gz" | sha256sum -c - \
- && tar -xvf geth.tar.gz \
- && mv geth-linux-amd64-1.15.11-36b2371c/geth /usr/local/bin/geth \
- && rm -rf geth.tar.gz geth-linux-amd64-1.15.11-36b2371c
+# Install Geth for the given architecture.
+RUN ARCH=$(dpkg --print-architecture) && \
+  echo "Detected architecture: $ARCH" && \
+  case "$ARCH" in \
+      "amd64") \
+          GETH_URL="https://gethstore.blob.core.windows.net/builds/geth-linux-amd64-1.15.11-36b2371c.tar.gz" && \
+          GETH_SHA="a14a4285daedf75ea04a7a298e6caa48d566a2786c93fc5e86ec2c5998c92455" && \
+          GETH_DIR="geth-linux-amd64-1.15.11-36b2371c" && \
+          VERIFY_SHA="true" \
+          ;; \
+      "arm64") \
+          GETH_URL="https://gethstore.blob.core.windows.net/builds/geth-linux-arm64-1.15.11-36b2371c.tar.gz" && \
+          GETH_SHA="148ec84db2268fa846ae68f6445f0c98d33e95069e40fe8c74b43ea5eb53df7b" && \
+          GETH_DIR="geth-linux-arm64-1.15.11-36b2371c" && \
+          VERIFY_SHA="true" \
+          ;; \
+      *) \
+          echo "Unsupported architecture: $ARCH" && exit 1 \
+          ;; \
+  esac && \
+  echo "Downloading: $GETH_URL" && \
+  curl -L "$GETH_URL" -o geth.tar.gz && \
+  echo "$GETH_SHA geth.tar.gz" | sha256sum -c -
```

Comment:
There is no continuation on this, and as a result it will result in an error when attempting to build this Docker image as the next line will be interpreted as a `Dockerfile` command that does not exist.


I assume this is meant to have a `&& \` at the end.